examples/src/proofs.rs

```diff
@@ -221,10 +221,13 @@ where
 /// Either print that the proof was successful or panic and return the error.
 #[inline]
 pub fn report_result(result: Result<(), impl Debug>) {
-    if let Err(e) = result {
-        panic!("{:?}", e);
-    } else {
-        println!("Proof Verified Successfully")
+    match result {
+        Err(e) => {
+            panic!("{:?}", e);
+        }
+        _ => {
+            println!("Proof Verified Successfully")
+        }
```

Comment:
Are you sure this `if let` needs to be replaced with a `match`? The `else` block doesn't depend on `result`, so it should be fine as it was, no?